### DISCUSSION TITLE
Airport PPM or CSFR passthrough like RF Design TXMOD

### DISCUSSION THREAD

**Discussion by Kieraneasdale** [ExpressLRS/ExpressLRS#2449] [Ideas]
Did anyone else remember watching this video from Mads Tech (https://youtu.be/1GVCLOvFTb4?si=g03tQgNczEDKyF_m) and thinking, THAT is a cool system!

But now we have airport I feel like we are one step closer to the end goal. Most Tx modules now have WiFi. Would it be possible to have a configuration setting to swap the standard CSFR telemetry to MAVLink with PPM or CSFR passthrough. 

The would eliminate the need for a RC link and TELEM link. Is there anything I can do to get things rolling for this feature and branch?

Thanks all!

**Comment by JyeSmith** [comment]
This is sort of where things are heading with the https://github.com/ExpressLRS/ExpressLRS/tree/mavlink-rc branch, and the backpack flashed with https://github.com/tridge/mavesp8266

Discord is the best place for help on this during active development. https://discord.com/channels/596350022191415318/1165186741716656230
